extensions/eda/plugins/event_source/aws_sqs_queue.py

```diff
@@ -31,6 +31,12 @@
     description:
       - Optional URL to connect to instead of the default AWS endpoints.
     type: str
+  queue_owner_aws_account_id:
+    description:
+      - Optional The Amazon Web Services account ID of the account that
+      - created the queue. This is only required when you are attempting
```

Comment:
Adding dash (`-`) creates a new paragraph in documentation. So we should just have one on top and remove the rest, unless we mean to add a paragraph.